packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -447,6 +453,19 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     const SingleActivator(LogicalKeyboardKey.arrowUp, shift: true, alt: true): const DoNothingAndStopPropagationTextIntent(),
   };
 
+  // Hand backspace/delete events that do not depend on text layout back to the
```

Comment:
This comment is a bit confusing, maybe add some more explanation why this is needed.

Also, would this apply to macos?

Reply:
macOS already does that (and the default shortcuts are already disabled)

Reply:
is that true? I saw the _macDisablingTextShortcuts is using _commonDisablingTextShortcuts which does not include delete and backward

Reply:
huh backspacing isn't deleting composing text on macOS

Reply:
added macOS delete/backspaces to the list. From the looks of it that fixes https://github.com/flutter/flutter/issues/111113

Reply:
The change seems to work fine with the default macOS chinese/japanese IME